### ISSUE TITLE
Add Assignment radio button labels

### ISSUE THREAD

**Issue by david-yz-liu** [MarkUsProject/Markus#1486]
From Dan Zingaro: it would be useful if the radio buttons for "Allow Web Submits" and "Display grader names" had correct labels. Right now, his screen reader only says "Yes" and "No" for them, which that isn't helpful. It should be noted that this problem doesn't occur on other radio buttons on the "Add Assignment" page.
